index.html

```diff
@@ -0,0 +1,154 @@
+                    <a href="index.html" class="logo-link">Code<span>G</span>ate</a>
+                </div>
+                <nav>
+                    <ul>
+                        <li class="single-nav active"><a href="about.html">About</a></li>
+                        <li class="single-nav">Program</li>
+                        <li class="single-nav">Join</li>
+                        <li class="single-nav">Sponsor</li>
+                        <li class="single-nav">News</li>
+                        <li class="single-nav square">CC Campaign</li>
+                    </ul>
+                </nav>
+            </div>
+
+        </header>
+        <div class="mobile-body">
+            <img id="menu-close" src="./images/close.png" alt="">
+            <ul class="nav-mobile">
+                <li><a href="index.html" class="nav-a">Home</a></li>
+                <hr>
+                <li><a href="about.html" class="nav-a">About</a></li>
+                <hr>
+                <li><a href="#!" class="nav-a">Join</a></li>
+                <hr>
+                <li><a href="#!" class="nav-a">Sponsor</a></li>
+                <hr>
+                <li><a href="#!" class="nav-a">News</a></li>
+
+            </ul>
+        </div>
+        <div class="main-content">
+            <q class="intro">Softwares With Energy</q>
+            <h1 class="main-into">We live and breathe software development</h1>
+            <p class="description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Ipsa odio illo tempora corporis quaerat sequi, nam repudiandae? Libero labore praesentium fugiat doloremque distinctio neque dolores odio nam deserunt maiores. Natus explicabo cupiditate deleniti recusandae eius tempore odio sunt, ipsum accusamus.</p>
+            <h3 class="time">2015.10.15(THU) ~ 16(FRI)</h3>
+            <P class="copy">@codegate of Rwanda, Art Center kigali and more</P>
+            
+        </div>
+    </section>
+    <section class="about">
+        <div class="about-title">
+            <h3>Main Program</h3>
+            <hr>
+        </div>
+        <div class="card-container">
+            <div class="card">
+                <div class="card-image">
+                    <img src="./images/django.png" alt="">
+                </div>
+                <div class="card-title">
+                    <h3>Django</h3>
+                </div>
+                <div class="card-detail">
+                    <p>we can develop your application by using django </p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-image">
+                    <img src="./images/react.png" alt="">
+                </div>
+                <div class="card-title">
+                    <h3>React</h3>
+                </div>
+                <div class="card-detail">
+                    <p>we can develop your application by using react </p>
+                </div>
+            </div>
```

Comment:
- [ ] I suggest that instead of using a `<div>` for each card, you use the `<article>` tag. This one represents a self-contained composition which is perfect to define a card! 🤓 🔧 

```suggestion
            <article class="card">
                <div class="card-image">
                    <img src="./images/django.png" alt="">
                </div>
                <div class="card-title">
                    <h3>Django</h3>
                </div>
                <div class="card-detail">
                    <p>we can develop your application by using django </p>
                </div>
            </div>
            <div class="card">
                <div class="card-image">
                    <img src="./images/react.png" alt="">
                </div>
                <div class="card-title">
                    <h3>React</h3>
                </div>
                <div class="card-detail">
                    <p>we can develop your application by using react </p>
                </div>
            </article>
```

- [x] Also, please include one more card to match the five present in the given design! 🤓 🔧